.github/workflows/ValidatePullRequest.yml

```diff
@@ -5,21 +5,47 @@ name: Validate Pull Request
 on:
   pull_request:
     branches: [main, "release/**"]
-    paths-ignore:
-      - '**.md'
-      - '**.txt'
   merge_group:
 
 permissions:
   id-token: write
   contents: read
 
 jobs:
+  docs-pr:
+    runs-on: ubuntu-latest
+    outputs:
+      not-just-docs: ${{ steps.not-just-docs.outputs.result }}
+    steps:
+      - uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          filters: |
+            docs:
+              - '**/*.md'
+              - '**/*.txt'
+            all:
+              - '**/*'
+      - uses: actions/github-script@v7
+        id: not-just-docs
+        with:
+          script: |
+            let docs_file_count = "${{steps.changes.outputs.docs_count}}";
+            let all_file_count = "${{steps.changes.outputs.all_count}}";
+            let result = (all_file_count - docs_file_count) > 0;
```

Comment:
```suggestion
            console.log(all_file_count > docs_file_count);
```

Reply:
Should also remove the line below, but the phone wouldn't let me do multiline selection